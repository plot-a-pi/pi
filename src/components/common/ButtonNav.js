import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Header.css';

const ButtonNav = () => {
  if(/teacher-sessions/.test(window.location.href))
    return (
      <div className={styles.sessionNav}>
        <Link to='/'>
          <button>Back to Home</button>
        </Link>
      </div>
    );
  return (
    <>
      <div className={styles.nav}>
        <Link to='/'>
          <button>Home Page</button>
        </Link>
        <Link to='/teachers'>
          <button>For Groups</button>
        </Link>
        <Link to='/montecarlo'>
          <button>Monte Carlo</button>
        </Link>
        <Link to='/submit-to-global'>
          <button>Plot A Pi</button>
        </Link>
      </div>
    </>
  );
};

export default ButtonNav;
