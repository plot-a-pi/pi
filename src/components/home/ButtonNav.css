.ButtonNav {
  margin-bottom: 20vh;
}