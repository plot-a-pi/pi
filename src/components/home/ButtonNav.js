import React from 'react';
import { Link } from 'react-router-dom';

const ButtonNav = () => (
  <>
    <Link to='/teachers'>
      <button>For Teachers</button>
    </Link>
    <Link to='/pifaqs'>
      <button>Pi FAQS</button>
    </Link>
    <Link to='/montecarlo'>
      <button>Monte Carlo</button>
    </Link>
    <Link to='/map'>
      <button>Data Map</button>
    </Link>
    <Link to='/submit-to-global'>
      <button>Submit to Global Data</button>
    </Link>
  </>
);

export default ButtonNav;
