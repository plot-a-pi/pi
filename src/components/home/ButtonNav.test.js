import React from 'react';
import { shallow } from 'enzyme';
import ButtonNav from './ButtonNav';

describe('ButtonNav Component', () => {
  it('should render a ButtonNav', () => {
    const wrapper = shallow(<ButtonNav/ >);
    expect(wrapper).toMatchSnapshot();
  });
});
