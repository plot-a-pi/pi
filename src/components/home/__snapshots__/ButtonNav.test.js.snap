// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonNav Component should render a ButtonNav 1`] = `
<Fragment>
  <Link
    to="/teachers"
  >
    <button>
      For Groups
    </button>
  </Link>
  <Link
    to="/pifaqs"
  >
    <button>
      Pi FAQS
    </button>
  </Link>
  <Link
    to="/montecarlo"
  >
    <button>
      Monte Carlo
    </button>
  </Link>
  <Link
    to="/map"
  >
    <button>
      Data Map
    </button>
  </Link>
  <Link
    to="/submit-to-global"
  >
    <button>
      Submit to Global Data
    </button>
  </Link>
</Fragment>
`;
